backend/common/models/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .auth import *
+from .mixins import *
```

Comment:
Poznamka k souboru `mixins.py` -- nechal bych ten kod v `commons/models.py`. Neni to sice klasicky model, ale svou funkcionalitou primo s modely souvisi. Mozna ze casem budeme mit potrebu napsat nejaky mixin pro views, a pak se nam to bude plest (co je obsahem `commons/views.py`? mixiny pro modely? mixiny pro view? pro oboji?)

To ze to je neco obecneho urcuje uz samotna aplikace `common/` kam budeme davat obecnosti uzivane vsude okolo.

Reply:
Souhlasím :) Není ještě potřeba to takhle dělit.